source/docs/magento2/advanced.md

```diff
@@ -14,3 +14,11 @@ _Since version 2.7.0_
 Front-Commerce allows you to send additional headers in all API calls. To do so, you must define the `magento.api.extraHeaders` (for storefront API) and/or `magento.api.extraAdminHeaders` (for admin API) configuration values from a [configuration provider](/docs/advanced/server/configurations.html#What-is-a-configuration-provider).
 
 These additional headers could be useful if you want to add additional context to your queries, depending on the request or to detect Front-Commerce requests from your Magento server.
+
+## Clearing Front-Commerce cache
+
+To clear the front commerce cache, the below command needs to be run on the magento server:
```

Comment:
The admin way (using the Front-Commerce cache from the caching page in admin) should also be documented here

Reply:
done [here](f884d745662713a8572b9f2c45b72009631ec4d6)